camelot/view/action_steps/application.py

```diff
@@ -240,7 +246,7 @@ def gui_run(self, gui_context, serialized_step):
         menu_bar.setCornerWidget(BusyWidget())
 
 @dataclass
-class InstallTranslator(ActionStep):
+class InstallTranslator(ActionStep, DataclassSerializable):
```

Comment:
@jeffh92 : tbd : hoe deze step eigenlijk zou moeten werken in een client-server verhaal ...